src/utils/slices.js

```diff
@@ -98,3 +100,32 @@ export function compareSlicesByTime(a, b, orderDir = "asc") {
   if (Number.isNaN(bmin)) return -direction;
   return amin > bmin ? direction : -direction;
 }
+
+export function isChange(path) {
```

Comment:
@dhruvbansal curious what you think of this utility. I was thinking there's a chance it would make sense in some form in unchained-bitcoin, especially the regexes which seem like they could be useful, _however_ opted against because of the uncertainty around the "prefix" which is kind of UI/wallet specific, as has been discussed elsewhere. I made a constant for the prefix so we can hopefully change that easily in the future, and other than that, this should be pretty flexible. You can see the tests for the kind of things it filters for.

Reply:
I think we need a concept of braid in `unchained-bitcoin` before we can have a concept of change.  I'm good with this code living here for now.